hook!-full-game/src/Areas/SpikeLevelTest.tscn

```diff
@@ -0,0 +1,125 @@
+[gd_scene load_steps=10 format=2]
```

Comment:
You added a spike to LevelTest so we don't need this extra scene

Reply:
Ohh sorry, it wasn't meant to commit the current `LevelTest.tscn` :sweat_smile: I duplicated it with the Spikes so it wouldn't cause conflicts. It was meant to push only the `SpikeLevelTest.tscn`